ccd-definition/CaseField.json

```diff
@@ -245,6 +245,55 @@
     "FieldType": "AllocationProposal",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "allocationDecision_header",
+    "Label": "## Allocation decision",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "allocationDecision",
+    "Label": " ",
+    "FieldType": "AllocationProposal",
+    "SecurityClassification": "Public",
+    "ShowSummaryContentOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "allocationProposalPresent",
+    "Label": "allocation proposal has been entered",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "missingAllocationDecision_Label",
+    "Label": "The local authority has not made an allocation proposal.",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "allocationDecision_Label",
+    "Label": "The proposed allocation is ${allocationProposal.proposal}.",
```

Comment:
Is this supposed to be above other label? 
![Screenshot 2019-09-13 at 10 35 53](https://user-images.githubusercontent.com/42037338/64852942-48c2eb80-d612-11e9-9c53-ff37dc9d684f.png)

Reply:
Isn't it? Not sure what do you mean by your comment (sorry too many labels :P) 

We have got either:
```
The proposed allocation is District judge

Reasoning behind district judge. 
```
Or
```
The local authority has not made an allocation proposal
```

Above that we have got  Allocation decision + case number + allocation decision
and below Which level of judge is needed for thise case.

Reply:
Yes but the proposed allocation comment is above in the screenshots so just for consistency I would move these two around ? :)

Reply:
Ah yes in file! Yes of course :)